operator/cmd/mesh/uninstall.go

```diff
@@ -135,12 +139,30 @@ func uninstall(cmd *cobra.Command, rootArgs *RootArgs, uiArgs *uninstallArgs, lo
 		l.LogAndFatal(err)
 	}
 
-	if uiArgs.revision != "" {
-		revisions, err := tag.ListRevisionDescriptions(kubeClient)
+	if uiArgs.revision != "" && uiArgs.revision != "default" {
+		iopCRs, err := kubeClient.Dynamic().Resource(apimachinery_schema.GroupVersionResource{
```

Comment:
this's fine now, but I recall there's a proposal from @howardjohn which may remove installed stated.

Reply:
IMO a missing webhook should not block the uninstall process, even though this might be an uncommon case. This PR is an attempt to fix this issue. I considered that proposal when I wrote this PR, but since I'm not clear about the state of that proposal, I'm not sure whether incorporating this logic at this time is good or not. Alternatively, we could remove the webhook detection logic entirely. Since it may not be 100% accurate, as the webhook is merely a resource installed during the installation process.